src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Kristian Gunder Kramås, Anders Mathiesen'
+__email__ = 'kristiakr@nmbu.no, andermat@nmbu.no'
+
+
+def single_game(num_players):
+    """
+        Returns duration of single game.
+
+        Arguments
+        ---------
+        num_players : int
+            Number of players in the game
+
+        Returns
+        -------
+        num_moves : int
+            Number of moves the winning player needed to reach the goal
+        """
+    import random
+    ladders = {1: 39, 8: 2, 36: 16, 43: 19, 49: 30, 65: 17, 68: 14}
+    snakes = {24: -19, 33: -30, 42: -12, 56: -19, 64: -37, 74: -62, 87: -17}
+    throws_all_players = []
+    for _ in range(num_players):
+        throws = 0
+        place = 0
+        while place < 90:
+            place += random.randint(1, 6)
+            if place in ladders:
```

Comment:
to avoid doing two seperate `if` tests, you could cuncatinate the dictionaries and also use the `.get()` method using 0 as the `default` value. example:
```
change = {**snakes, **ladders}.get(current_pos, default=0)
position += change
```